addons/mail/tests/test_mail_composer.py

```diff
@@ -261,3 +264,66 @@ def test_mail_mass_mode_compose_with_mso(self):
             values[self.partner_employee.id]['body_html'],
             'We must preserve (mso) comments in email html'
         )
+
+    @users('employee')
+    def test_mail_attachment_to_download_link(self):
```

Comment:
Maybe we could could also make a bit more low-level tests in "test_mail_mail" file ? Notably checking that we take either the server params, either a config parameter if not set / if we have the ICP, ...

We can keep this test as the "higher level one", indicating it works in notification process.